src/go/k8s/controllers/redpanda/console_controller.go

```diff
@@ -101,7 +101,22 @@ func (r *ConsoleReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ct
 }
 
 func (r *ConsoleReconciler) createIngress(ctx context.Context, cluster *redpandav1alpha1.Cluster, console *redpandav1alpha1.Console, log logr.Logger) error {
-	return nil
+	subdomain := ""
+	proxyAPIExternal := cluster.PandaproxyAPIExternal()
+	if proxyAPIExternal != nil {
+		subdomain = fmt.Sprintf("%s.%s", "console", proxyAPIExternal.External.Subdomain)
+	}
+	// If subdomain is empty, no Ingress will be created.
```

Comment:
My personal feeling is that we should have debug info that if subdomain is empty the ingress will not be created. In current implementation this knowledge what is available and what is not is hardcoded and undocumented.

Reply:
Added log in `pkg/resources/ingress.go`